### ISSUE TITLE
Your Blog Link on the Front Page is Broken

### ISSUE THREAD

**Issue by machineghost** [tastejs/todomvc#1893]
When you click on the "Blog" link on the front page (http://todomvc.com/) it takes you to a 404 page on Medium.

**Comment by manaswinidas** [comment]
Yes, I came across it too. Are you at it? I'm interested to fix it.
